modulefiles/wcoss_cray/fv3

```diff
@@ -31,23 +32,27 @@ module load gcc/4.9.2
 module load jasper-gnu-sandybridge/1.900.1
 module load png-intel-sandybridge/1.2.49
 module load zlib-intel-sandybridge/1.2.7
-module load crtm-intel/2.2.6
+setenv PNG_ROOT /usrx/local/prod//png/1.2.49/intel/sandybridge
 
-module use /gpfs/hps/usrx/local/nceplibs/NCEPLIBS/modulefiles
-module load g2/3.1.1
-module load g2tmpl/1.6.0
 ##
-## load ncelibs libraries
+## NCEP libraries
 ##
-module load bacio/2.0.3
-module load ip/3.0.2
-module load sp/2.0.3
-module load w3nco/2.0.7
-module load w3emc/2.3.1
-module load nemsio/2.2.4
+module use /usrx/local/nceplibs/NCEPLIBS/cmake/install/NCEPLIBS_UFS/modules
+module load bacio/2.4.0
+module load crtm/2.3.0
+module load g2/3.4.0
+module load g2tmpl/1.9.0
+module load ip/3.3.0
+module load nceppost/dceca26
+module load nemsio/2.5.1
+module load sp/2.3.0
+module load w3emc/2.7.0
+module load w3nco/2.4.0
+
+module load gfsio/1.4.0
+module load sfcio/1.4.0
+module load sigio/2.3.0
```

Comment:
May I ask not to load gfsio, sfcio and sigio? Model does not need those extra old libraries? Also what is nceppost/dceca26, why do we need to add them to model module file?

Reply:
nceppost/dceca26 is post library.